paddle/fluid/framework/parallel_executor.cc

```diff
@@ -63,13 +63,14 @@ ParallelExecutor::ParallelExecutor(
   // Step 1. Bcast the params to devs.
   // Create local scopes
   if (local_scopes.empty()) {
-    for (size_t i = 0; i < member_->places_.size(); ++i) {
-      member_->local_scopes_.push_back(&scope->NewScope());
+    member_->local_scopes_.emplace_back(member_->global_scope_);
```

Comment:
Comments needed here to indicate that the `global_scope` and `local_scope 1` share the same object.